### PR TITLE
Update README with more cyclone info

### PR DESCRIPTION
**Pull request by nsorros**
# Description

Adds more information about how to run jobs in the Cyclone super computer in batch
and interactive mode

# Checklist

- [ ] Add link to BigBoard issue
- [ ] Add tests for new functionality
- [x] Update README if necessary

### CODE REVIEW

**Review by chr7stos — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -21,20 +21,65 @@ Cyclone is the name of the super computer at the Cyprus Institute. In order to r
...
 ```
 
-To queue a job for training run
+You can run your first batch job by running
 ```
 sbatch job.sub
 ```
+you can see the output of the logs in `train.log`
+
+The super has two modes for running jobs
```
Comment (pedroandreou) [inline README.md:40]: the super computer*

File: README.md [hunk 2]
```diff
@@ -21,20 +21,65 @@ Cyclone is the name of the super computer at the Cyprus Institute. In order to r
...
+
+- interactive
+- batch
+
+## Interactive
+
+In the interactive model, you connect to a node and run a job by invoking a script as you would normally do. You can connect
```
Comment (pedroandreou) [inline README.md:47]: interactive mode*

File: README.md [hunk 3]
```diff
@@ -21,20 +21,65 @@ Cyclone is the name of the super computer at the Cyprus Institute. In order to r
...
+to a node using `salloc`, for example
+```
+salloc --nodes=1 --ntasks-per-node=8 --mem=8000
+```
+this command connects you to 1 node with 8 cpu codes and 8GB of memory.
+
+Then you can a python script for example,
```
Comment (pedroandreou) [inline README.md:54]: you can run*

**Review by pedroandreou — APPROVED** [review]
LGTM!

**Review by ivyleavedtoadflax — APPROVED** [review]

**Pull request merged**